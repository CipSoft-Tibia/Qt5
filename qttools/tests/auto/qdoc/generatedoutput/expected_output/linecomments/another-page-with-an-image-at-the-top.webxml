<?xml version="1.0" encoding="UTF-8"?>
<WebXML>
    <document>
        <page name="another-page-with-an-image-at-the-top.html" href="another-page-with-an-image-at-the-top.html" status="active" location="line_comment_adventures.qdoc" documented="true" subtype="page" title="This page starts with an image" fulltitle="This page starts with an image" subtitle="" brief="This page has an image right at the top">
            <description>
                <brief>This page has an image right at the top.</brief>
                <image href="images/leonardo-da-vinci.png"/>
                <para>The image should render as expected, and the alternate text should be there, too. This text is contained in its own paragraph following the image.</para>
                <para>This paragraph is a new paragraph, and doesn't contain an image.</para>
            </description>
        </page>
    </document>
</WebXML>
