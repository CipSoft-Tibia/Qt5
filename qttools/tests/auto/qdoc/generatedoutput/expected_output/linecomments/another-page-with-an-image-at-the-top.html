<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
<!-- line_comment_adventures.qdoc -->
  <meta name="description" content="This page has an image right at the top.">
  <title>This page starts with an image | LineComment</title>
</head>
<body>
<div class="sidebar"><div class="sidebar-content" id="sidebar-content"></div></div>
<h1 class="title">This page starts with an image</h1>
<!-- $$$another-page-with-an-image-at-the-top.html-description -->
<div class="descr" id="details">
<p class="centerAlign"><img src="images/leonardo-da-vinci.png" alt="This is the alternate text for the image" /></p><p>The image should render as expected, and the alternate text should be there, too. This text is contained in its own paragraph following the image.</p>
<p>This paragraph is a new paragraph, and doesn't contain an image.</p>
</div>
<!-- @@@another-page-with-an-image-at-the-top.html -->
</body>
</html>
